modules/tfm/tfm/boards/common/tfm_hal_platform.c

```diff
@@ -44,8 +44,7 @@ static enum tfm_hal_status_t crypto_platform_init(void)
 		return TFM_HAL_ERROR_BAD_STATE;
 	}
 
-#if CRYPTO_KEY_DERIVATION_MODULE_ENABLED && \
-    !defined(PLATFORM_DEFAULT_CRYPTO_KEYS)
+#ifdef CONFIG_HW_UNIQUE_KEY_RANDOM
 	if (!hw_unique_key_are_any_written()) {
 		SPMLOG_INFMSG("Writing random Hardware Unique Keys to the KMU.\r\n");
 		hw_unique_key_write_random();
```

Comment:
FYI this needs to be updated now with the return code.  hw_unique_key_write_random will now return an error code.

Reply:
You already did, rebase handled the rest.